V2/server/middleware/userauth.js

```diff
@@ -25,8 +25,8 @@ const verifyUser = async (req, res, next) => {
     }
     next();
   } catch (error) {
-    return res.status(404).send({
-      status: 404,
+    return res.status(400).send({
+      status: 400,
```

Comment:
The status code is wrong, refer to this article for the status codes https://developer.mozilla.org/en-US/docs/Web/HTTP/Status